codersdk/deployment.go

```diff
@@ -1265,6 +1306,29 @@ when required by your organization's security policy.`,
 			Group:         &deploymentGroupConfig,
 			Value:         &c.Config,
 		},
+		{
+			Name:        "CLI SSH Deployment Name",
+			Description: "The CLI SSH deployment name is the used in the Hostname of the ssh config.",
+			Flag:        "cli-ssh-deployment-name",
+			Env:         "CLI_SSH_DEPLOYMENT_NAME",
+			YAML:        "cliSSHDeploymentName",
+			Group:       &deploymentGroupClient,
+			Value:       &c.CLISSH.DeploymentName,
+			Hidden:      false,
+			Default:     "coder",
+		},
+		{
+			Name: "CLI SSH Config Options",
+			Description: "These cli config options will override the default ssh config options. " +
+				"Provide options in key=value format separated by commas." +
+				"Using this incorrectly can break ssh to your deployment. Use cautiously.",
+			Flag:   "cli-ssh-options",
+			Env:    "CLI_SSH_OPTIONS",
```

Comment:
Because this works with the VS Code extension, maybe we call this `CODER_SSH_CONFIG_OPTIONS`?

Reply:
True